src/arduino/index.ts

```diff
@@ -120,6 +120,7 @@ export class ArduinoVS {
         this.diagnostics.clear();
         this.output.clear();
         this.output.append('============== Begin to compile. ==============\n')
+        this.output.append(`${this.config.buildArgs.join(' ')}\n`);
```

Comment:
I'm considering add a verbose option to output this build/upload args, always showing or hiding may not suite everyone.